pkg/data/management/add.go

```diff
@@ -10,10 +10,15 @@ import (
 
 func Add(ctx context.Context, wrangler *wrangler.Context, management *config.ManagementContext) error {
 	_, err := addRoles(wrangler, management)
+
 	if err != nil {
 		return err
 	}
 
+	if err := addKEv2OperatorSchemas(management); err != nil {
```

Comment:
The function name says `addingOperatorSchemas` but I don't see any addition of schemas inside the function.

Reply:
okay. I think `syncOperatorDriverActiveState` would make sense.